_pages/welcome-to-TTS/classes/gsa-internal-tools.md

```diff
@@ -2,66 +2,31 @@
 title: GSA internal tools
 ---
 
-_This post provides information on how to work with GSA&rsquo;s web tools and your GSA-issued equipment._
+_This post provides information on how to work with GSA's web tools and your GSA-issued equipment._
```

Comment:
I applaud the person who took the time to put in a typographically correct apostrophe and am saddened that we are losing it.

Reply:
My thinking is that HTML encoding won't be something that many Handbook editors are familiar with, and thus it would be confusing to them. Note that we can convert them automatically using the `smartify` [filter](https://jekyllrb.com/docs/liquid/filters/) if we want.